### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by YOGARH**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by hbf1986 — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (YOGARH) [inline index.html:50]: Are emojis necessary?

**Review by YOGARH — COMMENTED** [review]
Have small comments.

File: index.html (hunk 1 above)
Comment (hbf1986) [inline index.html:50]:
```suggestion
<h2 hidden>Game over. Refresh to play again.!!</h2>
```

**Pull request merged**